app/src/main/res/menu/subscriptions.xml

```diff
@@ -1,35 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
-      xmlns:custom="http://schemas.android.com/apk/res-auto">
+    xmlns:custom="http://schemas.android.com/apk/res-auto">
 
     <item
-            android:id="@+id/refresh_item"
-            android:title="@string/refresh_label"
-            android:menuCategory="container"
-            custom:showAsAction="always"
-            android:icon="?attr/navigation_refresh"/>
+        android:id="@+id/refresh_item"
+        android:icon="?attr/navigation_refresh"
+        android:menuCategory="container"
+        android:title="@string/refresh_label"
+        custom:showAsAction="always" />
 
     <item
-            android:id="@+id/subscription_num_columns"
-            android:title="@string/subscription_num_columns"
-            custom:showAsAction="never">
+        android:id="@+id/subscriptions_filter"
+        android:title="@string/filter"
+        custom:showAsAction="never" />
+
+    <item
+        android:id="@+id/subscription_num_columns"
+        android:title="@string/subscription_num_columns"
+        custom:showAsAction="never">
         <menu>
             <item
-                    android:id="@+id/subscription_num_columns_2"
-                    android:checkable="true"
-                    android:title="2"/>
+                android:id="@+id/subscription_num_columns_2"
+                android:checkable="true"
+                android:title="2" />
```

Comment:
I would prefer to not reformat unrelated code because this leads to merge conflicts and makes `git blame` harder to read. We do not have a consistent code style for xml files that is checked on CI, so xml files get reformatted back and forth.

Reply:
I tried to revert this but these changes were made through android studios auto reformat and if I copy the code from the old files and paste them again they get automatically reformatted :sweat_smile: